model.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+class BmxResponse:
+    _links: dict
+    askAgainAfter: int
+    bmx_services: List
+
+class Link:
+    def __init__(self, href):
+        self.href = href
+    href: str
+
+class IconSet:
+    def __init__(self, defaultAlbumArt, largeSvg, monochromePng, monochromeSvg, smallSvg):
+        self.defaultAlbumArt = defaultAlbumArt
+        self.largeSvg = largeSvg
+        self.monochromePng = monochromePng
+        self.monochromeSvg = monochromeSvg
+        self.smallSvg = smallSvg
+    defaultAlbumArt: str
+    largeSvg: str
+    monochromePng: str
+    monochromeSvg: str
+    smallSvg: str
+
+class Asset:
```

Comment:
maybe we'll rename a lot of these classes for clarity, but it will become clear as we rigt